packages/react-ui-components/src/Icon/mapper.js

```diff
@@ -0,0 +1,505 @@
+        'forumbee',
+        'foursquare',
+        'free-code-camp',
+        'ge',
+        'get-pocket',
+        'gg',
+        'gg-circle',
+        'git',
+        'git-square',
+        'github',
+        'github-alt',
+        'github-square',
+        'gitlab',
+        'gittip',
+        'glide',
+        'glide-g',
+        'google',
+        'google-plus',
+        'google-plus-circle',
+        'google-plus-official',
+        'google-plus-square',
+        'google-wallet',
+        'gratipay',
+        'grav',
+        'hacker-news',
+        'houzz',
+        'html5',
+        'imdb',
+        'instagram',
+        'internet-explorer',
+        'ioxhost',
+        'joomla',
+        'jsfiddle',
+        'lastfm',
+        'lastfm-square',
+        'leanpub',
+        'linkedin',
+        'linkedin-square',
+        'linode',
+        'linux',
+        'maxcdn',
+        'meanpath',
+        'medium',
+        'meetup',
+        'mixcloud',
+        'modx',
+        'odnoklassniki',
+        'odnoklassniki-square',
+        'opencart',
+        'openid',
+        'opera',
+        'optin-monster',
+        'pagelines',
+        'paypal',
+        'pied-piper',
+        'pied-piper-alt',
+        'pied-piper-pp',
+        'pinterest',
+        'pinterest-p',
+        'pinterest-square',
+        'product-hunt',
+        'qq',
+        'quora',
+        'ra',
+        'ravelry',
+        'rebel',
+        'reddit',
+        'reddit-alien',
+        'reddit-square',
+        'renren',
+        'resistance',
+        'safari',
+        'scribd',
+        'sellsy',
+        'share-alt',
+        'share-alt-square',
+        'shirtsinbulk',
+        'simplybuilt',
+        'skyatlas',
+        'skype',
+        'slack',
+        'slideshare',
+        'snapchat',
+        'snapchat-ghost',
+        'snapchat-square',
+        'soundcloud',
+        'spotify',
+        'stack-exchange',
+        'stack-overflow',
+        'steam',
+        'steam-square',
+        'stumbleupon',
+        'stumbleupon-circle',
+        'superpowers',
+        'telegram',
+        'tencent-weibo',
+        'themeisle',
+        'trello',
+        'tripadvisor',
+        'tumblr',
+        'tumblr-square',
+        'twitch',
+        'twitter',
+        'twitter-square',
+        'usb',
+        'viacoin',
+        'viadeo',
+        'viadeo-square',
+        'vimeo',
+        'vimeo-square',
+        'vine',
+        'vk',
+        'wechat',
+        'weibo',
+        'weixin',
+        'whatsapp',
+        'wikipedia-w',
+        'windows',
+        'wordpress',
+        'wpbeginner',
+        'wpexplorer',
+        'wpforms',
+        'xing',
+        'xing-square',
+        'y-combinator',
+        'y-combinator-square',
+        'yahoo',
+        'yc',
+        'yc-square',
+        'yelp',
+        'yoast',
+        'youtube',
+        'youtube-play',
+        'youtube-square'
+    ];
+
+    if (!icon.startsWith('icon-')) {
+        return icon;
```

Comment:
Wait, but why do you return icons with `icon-` prefix? I thought you should have stripped it, no?

Reply:
I return icons WITHOUT icon- prefix. Maybe those are third party icons or just something special. at least we cannot detect it here and need to take it as it is.

Reply:
Duh, sorry, missed the `!`